transformation/src/jade-schema/tables/experiment_activity.table.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "experiment_activity",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_started",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_ended",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_submitted",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "description",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "assay",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "audit_labels",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "audit_severities",
+      "datatype": "string",
+      "type": "repeated"
+    },
+    {
+      "name": "award",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "lab",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "status",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "submitted_by",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "biosample_ids",
```

Comment:
We had said offline that the experiment should really point at all of its enclosing activities. Did we decide against that because it'd be too complicated to set up?

Reply:
We talked in person about this on Friday and decided that having an experiment point to its generated files would be a good solution.